### DISCUSSION TITLE
Getting Started with Git Thread 🔶

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#51393] [Programming Help]
At the heart of GitHub is a free and open-source version control system (VCS) called Git. Git is responsible for everything GitHub-related that happens locally on your computer. $ denotes a paid resource. 

**Links**:

- [Git Guides](https://github.com/github/training-kit/tree/main/git-guides) 
- [Code School: Git Real](https://www.pluralsight.com/courses/code-school-git-real) 
- $ [Learn Enough Git to Be Dangerous ](https://www.learnenough.com/git) 
- [Oh Shit, Git!?!](https://ohshitgit.com/) / [Dangit, Git!?!](https://dangitgit.com/en) 
- [Intro to Git](https://www.youtube.com/watch?v=FA6q4vnHZ_Y) 

**Books**:

- Git for Humans - David Demaree
- Pro Git - Scott Chacon & Ben Straub
- Learn Git in a Month of Lunches - Rick Umali 

Share any links, tutorials, videos, and books that have helped you learn Git below - don’t forget to include if it is paid too!

**Comment by airtower-luna** [comment]
"Pro Git" is excellent, I strongly echo that recommendation! It's also available online, including translations into many languages (look on the left): https://git-scm.com/book/en/v2 :books: 

If you're just getting started, chapters 2 and 3 cover the most important parts of Git.

**Reply by queenofcorgis** [reply]
Thanks for expanding on that @airtower-luna and sharing your experience with it ❤️

**Reply by RZ7272** [reply]
Pro Git" is excellent, I strongly echo that recommendation! It's also available online, including translations into many languages (look on the left): https://git-scm.com/book/en/v2 books

If you're just getting started, chapters 2 and 3 cover the most important parts of Git.

**Reply by MOHRICH** [reply]
Thanks

**Reply by lilke3113** [reply]
Me helping Sri according back plz sir help

**Comment by Anasheh0201** [comment]
assembly.2.12.02Write a program that can display the following shapes:

Rectangle
Triangle
Diamond
The program will read an input from the user. The input is terminated with a period. So the program will continue to run until a period is read.

The data the program reads is as follows:

A shape command followed by parameters.

R stands for Rectangle. The parameters are height (rows) and width (columns)

T stands for Triangle. The parameters is base width

D stands for Diamond. The parameters 

Your program needs to read the input and based on the input do the following:

Print your name
Then print the name of the shape
Then the shape below it.
Sample input:
R 03 10
T 05
D 11
.

**Comment by RZ7272** [comment]
> "Pro Git" is excellent, I strongly echo that recommendation! It's also available online, including translations into many languages (look on the left): https://git-scm.com/book/en/v2 books
> 
> If you're just getting started, chapters 2 and 3 cover the most important parts of Git.

**Comment by tommy2scripts** [comment]
Lets git er done!

**Reply by tripseeofficial-lab** [reply]
Thanks for you

**Comment by Armando3888** [comment]
> At the heart of GitHub is a free and open-source version control system (VCS) called Git. Git is responsible for everything GitHub-related that happens locally on your computer. $ denotes a paid resource.
> 
> **Links**:
> 
> * [Git Guides](https://github.com/github/training-kit/tree/main/git-guides)
> * [Code School: Git Real](https://www.pluralsight.com/courses/code-school-git-real)
> * $ [Learn Enough Git to Be Dangerous ](https://www.learnenough.com/git)
> * [Oh Shit, Git!?!](https://ohshitgit.com/) / [Dangit, Git!?!](https://dangitgit.com/en)
> * [Intro to Git](https://www.youtube.com/watch?v=FA6q4vnHZ_Y)
> 
> **Books**:
> 
> * Git for Humans - David Demaree
> * Pro Git - Scott Chacon & Ben Straub
> * Learn Git in a Month of Lunches - Rick Umali
> 
> Share any links, tutorials, videos, and books that have helped you learn Git below - don’t forget to include if it is paid too!

**Comment by shahnawaz-hussaink** [comment]
How should we push or code in beginning?? Like privately or Publicly??

**Comment by junayet9078gimrelcom-wq** [comment]
> At the heart of GitHub is a free and open-source version control system (VCS) called Git. Git is responsible for everything GitHub-related that happens locally on your computer. $ denotes a paid resource.
> 
> **Links**:
> 
> * [Git Guides](https://github.com/github/training-kit/tree/main/git-guides)
> * [Code School: Git Real](https://www.pluralsight.com/courses/code-school-git-real)
> * $ [Learn Enough Git to Be Dangerous ](https://www.learnenough.com/git)
> * [Oh Shit, Git!?!](https://ohshitgit.com/) / [Dangit, Git!?!](https://dangitgit.com/en)
> * [Intro to Git](https://www.youtube.com/watch?v=FA6q4vnHZ_Y)
> 
> **Books**:
> 
> * Git for Humans - David Demaree
> * Pro Git - Scott Chacon & Ben Straub
> * Learn Git in a Month of Lunches - Rick Umali
> 
> Share any links, tutorials, videos, and books that have helped you learn Git below - don’t forget to include if it is paid too!

**Comment by junayet9078gimrelcom-wq** [comment]
Hi👀
